pwa/src/styling/design-tokens/gateway-ui-design-tokens.css

```diff
@@ -50,11 +53,23 @@
 
   /* Colors */
   --gateway-ui-color-white: #ffffff;
+  --gateway-ui-color-black: black;
+  --gateway-ui-color-darkgrey: darkgrey;
+  --gateway-ui-color-red: red;
+  --gateway-ui-color-dark-red: rgb(146, 2, 2);
+  --gateway-ui-color-green: green;
+  --gateway-ui-color-status-green: #69b090;
+  --gateway-ui-color-status-red: #dd3c49;
```

Comment:
A "status" is never (I don't think..) red.

It could be "danger" or "error", to which we then assign a color. 

The only colors we can actually add to the tokens would be black and white, maybe grey. 

Other than that, we should aim to make named tokens, like `--gateway-ui-color-error`.